torch/distributions/constraints.py

```diff
@@ -210,7 +230,9 @@ def check(self, value):
 real = _Real()
 positive = _GreaterThan(0)
 greater_than = _GreaterThan
+greater_equal = _GreaterEqual
```

Comment:
Not right now. Do you suggest that I should remove them as of now?  I just wanted to fix the constraints of `greater_than` and `less_than`, otherwise the tests were failing for some distributions e.g. Beta.

Reply:
Yes, I think it's best to remove them. Thanks for fixing the `Beta` tests!